packages/modular-scripts/src/config/jest/index.ts

```diff
@@ -0,0 +1,116 @@
+import * as fs from 'fs-extra';
+import path from 'path';
+import chalk from 'chalk';
+import type { Config } from '@jest/types';
+import { getCracoJestConfig } from '../index';
+import getModularRoot from '../../utils/getModularRoot';
+import { ModularPackageJson } from '../../utils/isModularType';
+
+const modularRoot = getModularRoot();
+
+const rootPackageJson = fs.readJSONSync(
+  path.join(modularRoot, 'package.json'),
+) as ModularPackageJson;
+
+const packageJsonJest = rootPackageJson?.jest as
+  | Config.InitialOptions
+  | undefined;
+
+const jestConfigFile = fs.existsSync(path.join(modularRoot, 'jest.config.js'));
```

Comment:
Do we need to check for the existence of a jest config? I think if one exists it should be removed? What's happens with CRA if there's a ```jest.config``` file?